src/rest_api/classes/laboratory/widgets/alleles.clj

```diff
@@ -0,0 +1,42 @@
+(ns rest-api.classes.laboratory.widgets.alleles
+  (:require
+    [datomic.api :as d]
+    [clojure.string :as str]
+    [pseudoace.utils :as pace-utils]
+    [rest-api.formatters.object :as obj :refer [pack-obj]]))
+
+(defn alleles [laboratory]
+  (let [db (d/entity-db laboratory)
+        data (->> (d/q '[:find [?varent ...]
+                         :in $ ?laboratory
+                         :where 
+                         [?laboratory :laboratory/alleles ?varent] ]
+                       db (:db/id laboratory))
+                  (map (fn [wbvar]
+                         (let [allele (d/entity db wbvar)]
+                           (pace-utils/vmap
+                             :sequenced 
+                             (if (= (str (:variation/seqstatus allele))
+                                    ":variation.seqstatus/sequenced") "yes" "no")
+                             :type 
+                             (if (:method/id (:locatable/method allele))
+                               (-> (:method/id (:locatable/method allele))
+                                   (str/replace #"_allele" "")
+                                   (str/replace #"Allele" "")))
```

Comment:
No problem here, but could use `some->` to be more succinct:
```clojure
:type (some-> (:method/id (:locatable/method allele))
              (str/replace #"_allele" "")
              (str/replace #"Allele" ""))
```

Reply:
Oooh, that's really neat, thanks Matt !  I've made the changes, I appreciate the feedback while travelling =)